### ISSUE TITLE
Cannot access my Images

### ISSUE THREAD

**Issue by kev1807** [robcast/digilib#17]
After setting up tomcat and digilib properly (figured out, why tomcat wouldn't start after rebooting my Pi) and moving to another flat and proceeding with the History aspect of my work, I had a chance to continue my endeavours into image servers on a Raspberry Pi. 

I set it up again and have some issues, again. 
I think I am doing it right. 
I built digilib according to the Developer Build Instructions, since I wanted to play around (https://robcast.github.io/digilib/build-maven.html) 

From thence, I wandered to changing the digilib-config.xml 
I decided to make a basic config and move step by step 

So I changed the "basedir-list value to "/home/pi/Pictures" as I put some images there for testing purposes. 

All I get, however, is stunningly not the images I put there: 
![grafik](https://user-images.githubusercontent.com/15955790/49045230-9cbe2a00-f1d0-11e8-962e-0e222962e704.png)

My ideas are that somehow, the digilib cannot access the directory, so I moved the images to /opt/tomcat/webapps/digilib/img and adjusted the digilib-config.xml to make it right - to no avail. The result stays the same (After all, I created a user (and group) tomcat with home/dir in /opt/tomcat and made them the sole owners, so my idea was -well - obvious!) 

And, since I have read for hours, I wanted you to take a look at my result for http://192.168.178.104:8080/digilib/server/dlConfig.jsp (does not work from the Pi directly, I tried!)
But I get only this:
![grafik](https://user-images.githubusercontent.com/15955790/49045546-a1cfa900-f1d1-11e8-99ac-3f7f3e36f2a2.png) 

Maybe it is due to the late night, but I encountered this, when I first changed the digilib-config.xml about two hours back. It's frustrating, since I'm quite sure, that I am doing something wrong... 

To sum it up: 
digilib runs smoothly on Raspberry Pi B3. And I am doing something wrong. 

Upon returning to the "digilib-config.xml.template as my config-file, I get for all pages.
![grafik](https://user-images.githubusercontent.com/15955790/49046010-0b9c8280-f1d3-11e8-8af7-11007babb5a1.png)

However, the Global servlet configuration page is accessible once again.

Can I get some guidance? 

Thanks for your time and effort!
